CHANGELOG.md

```diff
@@ -1,3 +1,4 @@
+- Fixed Adding Base Media to Custom Channels for SUSE Liberty Linux in Client Configuration guide
```

Comment:
```suggestion
- Fixed Adding Base Media to Custom Channels for SUSE Liberty Linux in
  Client Configuration Guide
```
We'll copy this 1:1 into the package's `.changes` file; thus line breaks here.

Reply:
Thanks. Fixed.